src/ParallelAlgorithms/DiscontinuousGalerkin/InitializeMortars.hpp

```diff
@@ -0,0 +1,179 @@
+                    const Parallel::ConstGlobalCache<Metavariables>& /*cache*/,
+                    const ArrayIndex& /*array_index*/, ActionList /*meta*/,
+                    const ParallelComponent* const /*meta*/) noexcept {
+    using simple_tags =
+        db::AddSimpleTags<mortar_data_tag,
+                          ::Tags::Mortars<::Tags::Next<temporal_id_tag>, dim>,
+                          ::Tags::Mortars<::Tags::Mesh<dim - 1>, dim>,
+                          ::Tags::Mortars<::Tags::MortarSize<dim - 1>, dim>>;
+    using compute_tags = db::AddComputeTags<>;
+
+    const auto& element = db::get<::Tags::Element<dim>>(box);
+    const auto& temporal_id = get<temporal_id_tag>(box);
+    const auto& initial_extents = db::get<::Tags::InitialExtents<dim>>(box);
+
+    db::item_type<mortar_data_tag> mortar_data{};
+    db::item_type<::Tags::Mortars<::Tags::Next<temporal_id_tag>, dim>>
+        mortar_next_temporal_ids{};
+    db::item_type<::Tags::Mortars<::Tags::Mesh<dim - 1>, dim>> mortar_meshes{};
+    db::item_type<::Tags::Mortars<::Tags::MortarSize<dim - 1>, dim>>
+        mortar_sizes{};
+    const auto& interface_meshes =
+        db::get<::Tags::Interface<::Tags::InternalDirections<dim>,
+                                  ::Tags::Mesh<dim - 1>>>(box);
+    for (const auto& direction_and_neighbors : element.neighbors()) {
+      const auto& direction = direction_and_neighbors.first;
+      const auto& neighbors = direction_and_neighbors.second;
+      for (const auto& neighbor : neighbors) {
+        const auto mortar_id = std::make_pair(direction, neighbor);
+        mortar_data[mortar_id];  // Default initialize data
+        mortar_next_temporal_ids.insert({mortar_id, temporal_id});
+        mortar_meshes.emplace(
+            mortar_id,
+            dg::mortar_mesh(
+                interface_meshes.at(direction),
+                domain::Initialization::create_initial_mesh(
+                    initial_extents, neighbor, neighbors.orientation())
+                    .slice_away(direction.dimension())));
+        mortar_sizes.emplace(
+            mortar_id,
+            dg::mortar_size(element.id(), neighbor, direction.dimension(),
+                            neighbors.orientation()));
+      }
+    }
+
+    if (AddFluxBoundaryConditionMortars) {
+      const auto& boundary_meshes =
+          db::get<::Tags::Interface<::Tags::BoundaryDirectionsInterior<dim>,
+                                    ::Tags::Mesh<dim - 1>>>(box);
+      for (const auto& direction : element.external_boundaries()) {
+        const auto mortar_id =
+            std::make_pair(direction, ElementId<dim>::external_boundary_id());
+        mortar_data[mortar_id];
+        // Since no communication needs to happen for boundary conditions,
+        // the temporal id is not advanced on the boundary, so we set it equal
+        // to the current temporal id in the element
+        mortar_next_temporal_ids.insert({mortar_id, temporal_id});
```

Comment:
I probably knew this once, but are these actually used?  It sounds from the comment like they don't contain valid data, so I wonder if we can get rid of them.  The only use I immediately find checks if it is looking at an external boundary and ignores the entry if so.

Reply:
looks like these entries are indeed unused, so remove them.